### PR TITLE
Update Span Operations for Audit

### PR DESCRIPTION
**Pull request by AbhiPrasad**
https://www.notion.so/sentry/Set-up-an-audit-for-SDK-consistency-for-span-operations-to-enable-performance-issues-addf02a8fa234dda8acf48d4ff9b8efb?d=b852bdd1e9914d2ca74c1052296081f3#d04af01302604881999b1bd29c837b1f

This PR attempts to align our span operations based on an audit we've taken on in the past 2 weeks. The reason we are trying to align our span operations is because they are required to generate consistent performance issues: https://github.com/getsentry/sentry/discussions/37083

In addition, to this PR, we've collected a list of span operations we'll need to update across the different SDKs: https://github.com/getsentry/develop/issues/693#issuecomment-1259423033

### CODE REVIEW

**Comment by cleptric** [comment]
@AbhiPrasad I talked to @stayallive, and we're happy with
```
laravel.handle -> middleware.handle
laravel.bootstrap -> app.bootstrap
laravel.autoload -> app.php.autoload
laravel.view -> view.render
```
for Laravel 👍

File: src/docs/sdk/performance/span-operations.mdx [hunk 1]
```diff
@@ -6,136 +6,259 @@ Span operations are a short code identifying the type of operation the span is m
...
-| queue    |               |             |
-|          | queue.process |             |
-| job      |               |             |
-|          | job.exec      |             |
-| celery   |               |             |
+| Category | Usage                  | Description |
+| -------- | ---------------------- | ----------- |
+| topic    |                        |             |
+|          | topic.send             |             |
+|          | topic.recieve          |             |
```
[the remark this answers is not in the document]
Comment (AbhiPrasad) [reply src/docs/sdk/performance/span-operations.mdx:224]: there is no SDK that uses this - so we can def fix it! Good catch, that's been around for a while 😅

[the remark this answers is not in the document]
Comment (gggritso) [reply src/docs/sdk/performance/span-operations.mdx:224]: Beautiful, thank you!

**Comment by gggritso** [comment]
@AbhiPrasad out of curiosity, why is the convention `category.framework.action` and not `category.action.framework` (e.g., `ui.react.update` vs. `ui.update.react`)? I'm only asking because if we wanted to filter for _all_ UI update ops (`ui.react.update`, `ui.vue.update`, etc.), it'd be easier to do  `op.startswith("ui.update")` than having to do wildcard matches on `ui.*.update`. That's just an example, I'm only wondering why the convention is what it is!

**Comment by AbhiPrasad** [comment]
> @AbhiPrasad out of curiosity, why is the convention `category.framework.action` and not `category.action.framework` (e.g., `ui.react.update` vs. `ui.update.react`)? I'm only asking because if we wanted to filter for _all_ UI update ops (`ui.react.update`, `ui.vue.update`, etc.), it'd be easier to do `op.startswith("ui.update")` than having to do wildcard matches on `ui.*.update`. That's just an example, I'm only wondering why the convention is what it is!

It's because the action for the `ui.framework.X` is dependent on the framework itself. For example, `init` is a lifecycle in Svelte, but not in Vue or React.

**Comment by gggritso** [comment]
@AbhiPrasad that's what I figured, thanks!

**Review by marandaneto — APPROVED** [review]
Thank you @AbhiPrasad and @jan-auer

**Comment by AbhiPrasad** [comment]
We're now merging this - we can continue to iterate on this as needed.

**Pull request merged**